KoreBuild-dotnet/build/_k-standard-goals.shade

```diff
@@ -111,7 +112,14 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
       .Where(f => Path.GetFileName(f).Equals("project.json", StringComparison.OrdinalIgnoreCase))
       .ToList();
 
-    projectFiles.ForEach(projectFile => DotnetBuild(projectFile, Configuration));
+    projectFiles.ForEach(projectFile =>
```

Comment:
Is the publish a perf thing? (We're already publishing as part of the xunit-test target).